guides/css.md

```diff
@@ -0,0 +1,18 @@
+# Alternatives to `$.css()`
+
+
+## Option 1:
+
+Use getComputedStyle instead.
```

Comment:
There is also `style`, which usually makes a lot more sense than `GCS`, like in your example:

`elem.style.color`

It also covers how to set css props, which `$.css` does as well